crates/vm/Cargo.toml

```diff
@@ -4,27 +4,36 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-ethereum_rust-core.workspace = true
-ethereum_rust-storage.workspace = true
-revm = { version = "10.0.0", features = [
+ethereum_rust-core = { path = "../common", default-features = false }
```

Comment:
If we use the workspace syntax, and we want to import a crate as a standalone package, we wouldn't be able to "`configure`" it with features since we would have like a "circular" dependecy.

For example(using `ethereun_rust-vm/Cargo.toml`):
```toml
# [...] 

[dependencies]
ethereum_rust-core = { path = "../common", default-features = false }
ethereum_rust-storage = { path = "../storage/store", default-features = false }

# [...] 

[features]
default = ["libmdbx", "c-kzg", "blst"]
l2 = []
c-kzg = ["revm/c-kzg"]
blst = ["revm/blst"]
libmdbx = ["ethereum_rust-storage/default", "ethereum_rust-core/libmdbx"]
```

The `workspace's` .toml sets the `libmdbx` feature for the `storage` crate. And if we use `ethereum_rust-storage` using the workspace syntax it compiles `libmdbx`.